CHANGELOG.md

```diff
@@ -20,9 +20,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Broken Merlin logo image on PyPI summary page.
 
 ### Removed
-- Dependencies outside the requirements directory
-- LLNL-specific material in the Makefile
-- `merlin-templates` cli command
+- Dependencies outside the requirements directory.
+- LLNL-specific material in the Makefile.
+- `merlin-templates` cli command (now shows deprecation message).
```

Comment:
This should be under a separate `### Deprecated` header

Reply:
Fixed